cinema/views.py

```diff
@@ -167,6 +188,25 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="date",
+                description="Filter by date",
+                required=False,
+                type=OpenApiTypes.DATE,
+            ),
+            OpenApiParameter(
+                name="movie",
+                description="Filter by movie",
+                required=False,
+                type={"type": "list", "items": {"type": "number"}},
```

Comment:
It accepts one number, not list of numbers

Reply:
why? it can be filtered by many numbers, not just one

Reply:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/65166203/229090901-25064eb7-e766-4ad5-b0d4-52116157e5d7.png">
if movie_id_str:
Just one number)))